### PR TITLE
DRILL-5772: Check UTF-8 support set by default via saffron.properties…

### PR DESCRIPTION
**Pull request by omalashevsky**
… file

### CODE REVIEW

File: framework/resources/Functional/values/saffron_encoding_DRILL_5772.q [hunk 1]
```diff
@@ -0,0 +1,5 @@
+drop table if exists encoding_test;
+create table encoding_test as select name, alias, description from (values('Василий Теркин', 'Вася', 'персонаж'), ('王維', '摩詰', '詩人'), ('Ибн Рушд', 'ابن رشد', 'философ'), ('James Cook', 'James', 'navigator')) as t(name, alias, description);
+select name from encoding_test where alias in ('Вася', '摩詰', 'ابن رشد');
+drop table if exists encoding_test;
+select 1 from (values(0));
```
Comment (prasadns14) [inline framework/resources/Functional/values/saffron_encoding_DRILL_5772.q]: @omalashevsky, why do we need the last select statement?

Comment (prasadns14) [inline framework/resources/Functional/values/saffron_encoding_DRILL_5772.q]: Did you run pre-commit tests?

Comment (cchang738) [inline framework/resources/Functional/values/saffron_encoding_DRILL_5772.q]: @omalashevsky You can mark the query that you want to verify result with "--@test". Doing it this way, you don't need the last query. I believe the README file explains it.  After correction, and rerun pre-commit, if it passes, you can merge the PR. +1. Thanks.

**Comment by omalashevsky** [comment]
@Chun Chang
I have changed the test to use @test annotation. Thanks for pointing out. 
Regarding Pre-commit tests run, since CV QA team currently does not have test cluster, I have run only tests from values test suite. Is it possible to run Pre-commit tests on my branch in SJ before the commit?

**Comment by cchang738** [comment]
@omalashevsky  I launched a pre-commit run for you. Will see how it goes.

**Pull request merged**